fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

```diff
@@ -4069,9 +4065,33 @@ public void testPatchInsertMementoNamespacePredicate() throws IOException {
         final HttpPatch patch = new HttpPatch(subjectURI);
         patch.addHeader(CONTENT_TYPE, "application/sparql-update");
         patch.setEntity(new StringEntity(
-            "INSERT { <>  <" + MEMENTO_NAMESPACE + "mementoDate" + "> \"Thu, 21 Jan 2010 00:09:40 GMT\". } WHERE {}"));
+            "INSERT { <>  <" + MEMENTO_NAMESPACE + "mementoDatetime" +
+            "> \"Thu, 21 Jan 2010 00:09:40 GMT\". } WHERE {}"));
         try (CloseableHttpResponse response = execute(patch)) {
-            assertEquals("Must not be able to POST RDF that contains a memento namespace predicate",
+            assertEquals("Must not be able to PATCH RDF that contains a memento namespace predicate",
+                         CONFLICT.getStatusCode(), getStatus(patch));
+        }
+    }
+
+    @Test
+    public void testPatchUpdateIndirectContainerServerManaged() throws IOException {
+        final String pid = getRandomUniqueId();
+        final String subjectURI = serverAddress + pid;
+        createObjectAndClose(pid);
+        final String indirectContainerPid = getRandomUniqueId();
+        final String indirectContainerURI = serverAddress + indirectContainerPid;
+        createObjectAndClose(indirectContainerPid, INDIRECT_CONTAINER_LINK_HEADER);
+        final HttpPatch patch = new HttpPatch(indirectContainerURI);
+        patch.addHeader(CONTENT_TYPE, "application/sparql-update");
+        patch.setEntity(new StringEntity(
+                "PREFIX ldp: <http://www.w3.org/ns/ldp#>\n" +
+                "PREFIX fedora: <http://fedora.info/definitions/v4/repository#>\n" +
+                "PREFIX ore: <http://www.openarchives.org/ore/terms/>\n" +
+                "INSERT { <> ldp:membershipResource <" + subjectURI + "> ;\n" +
+                    "ldp:hasMemberRelation fedora:createdBy ;\n" +
+                    "ldp:insertedContentRelation ore:proxyFor. } WHERE {}"));
+        try (CloseableHttpResponse response = execute(patch)) {
+            assertEquals("Must not be able to PATCH IndirectContainer updating Server Managed triples",
                          CONFLICT.getStatusCode(), getStatus(patch));
```

Comment:
I believe you want to be checking the status of the `response`... otherwise, you are re-invoking the `patch` request.